code/drasil-code/lib/Language/Drasil/Code/Imperative/Parameters.hs

```diff
@@ -55,13 +55,13 @@ getInputFormatIns = do
   let getIns :: Structure -> InputModule -> [CodeVarChunk]
       getIns Bundled Separated = [quantvar inParams]
       getIns _ _ = []
-  getParams "get_input" In $ quantvar inFileName : getIns (inStruct g) (inMod g)
+  getParams (genICFuncName GetInput) In $ quantvar inFileName : getIns (inStruct g) (inMod g)
```

Comment:
something to investigate: does `getParams` need its first argument to be a `String` or can the call to `genICFuncName` be delayed?